notebooks/AllSupportedTypes.ipynb

```diff
@@ -0,0 +1,193 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "688a1886",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "681a472ef5ed4688ba6aaff5c582fc65",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "VegaWidget()"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 9 µs, sys: 11 µs, total: 20 µs\n",
+      "Wall time: 22.9 µs\n"
+     ]
+    }
+   ],
+   "source": [
+    "spec_no_data = {\n",
+    "  \"$schema\": \"https://vega.github.io/schema/vega-lite/v5.json\",\n",
+    "  \"data\": {\"name\": \"data\"},\n",
+    "  \"mark\": \"bar\",\n",
+    "  \"encoding\": {\n",
+    "    \"x\": {\"aggregate\": \"sum\", \"field\": \"yield\"},\n",
+    "    \"y\": {\"field\": \"variety\"},\n",
+    "    \"color\": {\"field\": \"site\"}\n",
+    "  }\n",
+    "}\n",
+    "from vega.widget import VegaWidget\n",
+    "import requests\n",
+    "import json\n",
+    "req = requests.get(\"https://raw.githubusercontent.com/vega/vega/master/docs/data/barley.json\")\n",
```

Comment:
Let's use Vega-datasets. Either the python package or https://cdn.jsdelivr.net/npm/vega-datasets@2.2.0/data/barley.json so we don't accidentally break things in the future.